### DISCUSSION TITLE
How can I get iceutil37.dll

### DISCUSSION THREAD

**Discussion by Henry0528** [zeroc-ice/ice#1308] [Q&A] [answered]
I've downloaded the lastest version of ice 3.7.6 from visual studio 2017.While I can't find iceutil37.dll ,I used to use iceutil35.dll in my project,now l’ve upgrated my ice version from 3.5.1 to 3.7.6 and l need iceutil37.dll to replace the old iceutil35.dll

**Comment by pepone** [comment, marked as answer]
There is no IceUtil library with 3.7, you just need ice37.dll
